code-challenges.test.js

```diff
@@ -26,37 +26,171 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: "-7 is not divisible by three"
 
+// Create a test in Jest seeing the test have good failure because the function is undefined
+
+describe(`evenThree`,  () => {
+  it(`should return a string indicating if the number is divisible by three or not`, () => {
+    expect(evenThree(object1)).toEqual("15 is divisible by three");
+    expect(evenThree(object2)).toEqual("0 is divisible by three");
+    expect(evenThree(object3)).toEqual("-7 is not divisible by three");
+  })
+})
+
+
+// Function returns undefined 
+
+//  Output :  FAIL  ./code-challenges.test.js
+//   evenThree
+//   ✕ if the number inside the object is evenly divisible by three (1 ms)
+
+// ● evenThree › if the number inside the object is evenly divisible by three
+
+//   ReferenceError: evenThree is not defined
+
+// class Divide {
+//   constructor(number) {
+//     this.number = number;
+//   }
+
+//   answer() {
+//     return this.number % 3 === 0;
+//   }
+// }
+
+// const object1 = new Divide(15);
+// const object2 = new Divide(0);
+// const object3 = new Divide(-7);
+
+// const evenThree = (obj) => {
+  
+// };
+
 // b) Create the function that makes the test pass.
 
+// Make the test pass by completing the function
+//  PASS  ./code-challenges.test.js
+//   evenThree
+
+class Divide {
+    constructor(number) {
+      this.number = number;
+    }
+  
+    answer() {
+      return this.number % 3 === 0;
+    }
+  }
+  
+  const object1 = new Divide(15);
+  const object2 = new Divide(0);
+  const object3 = new Divide(-7);
+  
+  const evenThree = (obj) => {
+    if (obj.answer()) {
+      return `${obj.number} is divisible by three`;
+    } else {
+      return `${obj.number} is not divisible by three`;
+    }
+  };
+  
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
+// input: an array of words
+// output: a single array with all words from both original arrays capitalized
+// process: Create a function that takes an array as a parameter.  Set the function up to concat both given arrays into one single array. Will need to create another constand to facilitate the array merger
+//create another const that will iterate over the entire array using the .map method beginning at the 0 index using charAt, then by using word.toUpperCase to go through the entire array by not indicating a starting index and using .splice to return at the beginning of the array and slice to start at the beginning array and leaving the parenthisis blank to return the last index of the array
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+describe("capitalizeWords", () => {
+  it("should capitalize all words in the array", () => {
+    const result = capitalizeWords(randomNouns1.concat(randomNouns2));
+    expect(result).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database", "Chopsticks", "Mango"]);
+  });
+});
```

Comment:
Make sure you pay attention to the prompt and expected outputs. The function should be passed one of the given variable arrays at a time, not both arrays combined